docs/ops/config.rst

```diff
@@ -338,6 +338,15 @@ in other Django projects.
    reduced to "read-only" methods when using the ``ProductionReadOnly``
    configuration.
 
+.. envvar:: DJANGO_OIDC_USER_ENDPOINT
+
+   :default: ``https://auth.mozilla.auth0.com/userinfo``
+
+   URL where we sent access tokens received as an authorization bearer token.
+   This URL needs to match the OIDC domain used by the client to authenticate.
+   The endpoint URL is actually extracted from reading ``/.well-known/openid-configuration``
```

Comment:
I think this may be confusing, and a reader might think that Normandy will make the request to the well-known URL. Maybe something like "The value for this setting is usually listed in `/.well-known/openid-configuration` on the OIDC provider."?

Reply:
That's a much better phrase. Thanks.